CommunicationLayer/MessagingFramework/MessageDefines.cpp

```diff
@@ -2,31 +2,52 @@
 
 namespace
 {
-    const int KEY_DRIVER_CONTROL_LENGTH = 21;
-    const int DRIVER_CONTROL_DETAILS_LENGTH = 29;
-    const int FAULTS_LENGTH = 9;
-    const int BATTERY_DATA_LENGTH = 18;
-    const int CMU_DATA_LENGTH = 42;
+    const int BATTERY_DATA_LENGTH = 63;
+    const int BATTERY_FAULTS_DATA_LENGTH = 6;
+    const int CMU_DATA_LENGTH = 53;
+    const int DRIVER_CONTROLS_DATA_LENGTH = 13;
+    const int KEY_MOTOR_DATA_LENGTH = 46;
+    const int LIGHTS_DATA_LENGTH = 5;
+    const int MOTOR_DETAILS_DATA_LENGTH = 72;
+    const int MOTOR_FAULTS_DATA_LENGTH = 12;
+    const int MPPT_DATA_LENGTH = 13;
+    const int OTHER_DATA_LENGTH = 20;
```

Comment:
There was a miscalculation in the protocol. These were updated in the protocol and need to be +1 each.